packages/miniflare/src/workers/kv/namespace.worker.ts

```diff
@@ -73,6 +76,46 @@ function secondsToMillis(seconds: number): number {
 	return seconds * 1000;
 }
 
+async function processKeyValue(
+	obj: KeyValueEntry<unknown> | null,
+	type: string = "text",
+	withMetadata: boolean = false
```

Comment:
```suggestion
	withMetadata = false
```

Reply:
similar type could be `type: "text" | "json" = "text"`